### ISSUE TITLE
32-bit Linux server crashes on startup

### ISSUE THREAD

**Issue by theSarrum** [multitheftauto/mtasa-blue#3080]
### Describe the bug

As the title says, the latest currently available 32-bit Linux server crashes on startup.

### Steps to reproduce

Get the latest 32-bit Linux server and run it.

### Version

Server: v1.6-r21826-release (Debian 10, 32-bit)

### Additional context

The last version that worked without issues was 1.5.9-release-21415.
And all available (32-bit) versions on the nightly page have this issue.

[crashdump  backtrace r21826.zip_](https://upload.mtasa.com/u/868684799/crashdump%20%20backtrace%20r21826.zip_)

### Relevant log output

```shell
MTA:SA Server v1.6-r21826-release crash report.
2023-06-18 21:35
Caught 18 addresses ...

Backtrace:
#0 - ./core.so(+0x6781d) [0xb7bf781d]
#1 - ./core.so(+0x903f8) [0xb7c203f8]
#2 - ./core.so(+0x90842) [0xb7c20842]
#3 - linux-gate.so.1(__kernel_rt_sigreturn+0) [0xb7f50d78]
#4 - /lib/i386-linux-gnu/libc.so.6(cfree+0x24) [0xb7dba714]
#5 - /games/mta-1.5/22000/mta-server-22000(_ZdlPv+0x1c) [0x4bdb5c]
#6 - /games/mta-1.5/22000/mods/deathmatch/deathmatch.so(+0x1373ac) [0xb65043ac]
#7 - /games/mta-1.5/22000/mods/deathmatch/deathmatch.so(+0x137741) [0xb6504741]
#8 - /games/mta-1.5/22000/mods/deathmatch/deathmatch.so(+0x22020d) [0xb65ed20d]
#9 - /games/mta-1.5/22000/mods/deathmatch/deathmatch.so(+0x224411) [0xb65f1411]
#10 - /games/mta-1.5/22000/mods/deathmatch/deathmatch.so(+0x1db375) [0xb65a8375]
#11 - /games/mta-1.5/22000/mods/deathmatch/deathmatch.so(+0x121b52) [0xb64eeb52]
#12 - ./core.so(+0x69c40) [0xb7bf9c40]
#13 - ./core.so(+0x6cfdf) [0xb7bfcfdf]
#14 - ./core.so(Run+0x98) [0xb7c10918]
#15 - /games/mta-1.5/22000/mta-server-22000(main+0x2a1) [0x4bd141]
#16 - /lib/i386-linux-gnu/libc.so.6(__libc_start_main+0xf1) [0xb7d58b41]
#17 - /games/mta-1.5/22000/mta-server-22000(_start+0x35) [0x4bd7b5]
--------------------------------------------------------------------------------
```

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by botder** [comment]
Please test again with revision 21884

Fixed in 0bf8c649e172637db94a36122aaec07fec75b4c6 + 9e261e271ac9cea3d83d8e7198e3cbe8e80578ff

**Comment by theSarrum** [comment]
Yes, it works now, thanks botder.
